bundles/org.openhab.binding.epsonprojector/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -293,6 +301,10 @@
 		<item-type>String</item-type>
 		<label>Color Mode</label>
 		<description>Retrieve or Set the Color Mode</description>
+		<tags>
+			<tag>Control</tag>
+			<tag>Color</tag>
```

Comment:
Is it color or mode? From the description it looks like it will indirectly influence the color, but since it's through a mode setting, I'm wondering which tag to use? I guess it's similar to the fan speed vs. mode mentioned in #2496.

Reply:
Again questions don't help. Please propose something concrete.